apps/accounts/serializers/account/account.py

```diff
@@ -460,7 +460,10 @@ class Meta(AccountSerializer.Meta):
             'secret': {'write_only': False},
             'spec_info': {'label': _('Spec info')},
         }
-        exclude_backup_fields = ['passphrase', 'push_now', 'params']
+        exclude_backup_fields = [
+            'passphrase', 'push_now', 'params',
+            'spec_info', 'platform', 'auto_config'
+        ]
 
 
 class AccountHistorySerializer(serializers.ModelSerializer):
```

Comment:
The major changes noted here include:

- In addition to `exclude=['passphrase', ...']`, you also specify `exclude=['params', 'spec_info']`. This is a potential improvement because it keeps the backup fields (`params`, `spec_info`) from being included in all account history data, making them easier to filter out if needed.

To summarize, these changes should improve efficiency or clarity in your model's validation processes for excluding specific types of information during serialization operations.